File: 251004/Yelkin/4-5/TopicService/src/main/java/by/yelkin/TopicService/entity/Comment.java
package by.yelkin.TopicService.entity;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
@Table(name = "tbl_comment")
public class Comment {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "topic_id", nullable = false)
    private Topic topic;
    private String content;
}

File: 251003/Kukhatskavolets/microblog/src/main/kotlin/by/kukhatskavolets/entities/Comment.kt
package by.kukhatskavolets.entities

import jakarta.persistence.*

@Entity
@Table(name = "tbl_comment", schema = "public")
data class Comment(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long = 0,

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "tweetId", nullable = false)
    val tweet: Tweet = Tweet(),

    @Column(nullable = false, length = 2048)
    val content: String  = ""
)